### PR TITLE
[Tests/Filter/MvNCSDK2] Add failure test cases 

### PR DESCRIPTION
**Pull request by wooksong**
This patch adds failure test cases, which are based on the normal pipeline launch test case. Note that the current line coverage of tensor_filter_movidius_ncsdk2.c is 87.2%.

Signed-off-by: Wook Song <wook16.song@samsung.com>

### CODE REVIEW

**Comment by wooksong** [comment]
> Add `_n` postfix for negative test cases

Updated.

**Comment by wooksong** [comment]
#### V2
- As mentioned in https://github.com/nnsuite/nnstreamer/pull/2064#pullrequestreview-354187004, the names of negative test cases are changed to use the '_n' prefix.

File: tests/nnstreamer_filter_mvncsdk2/NCSDKTensorFilterTestHelper.cc [hunk 1]
```diff
@@ -60,6 +80,7 @@ NCSDKTensorFilterTestHelper::init (model_t model)
     this->mGraphHandle = nullptr;
   }
   this->mModelPath = nullptr;
+  this->mFailStage =fail_stage_t::NONE;
```
Comment (dongju-chae) [inline tests/nnstreamer_filter_mvncsdk2/NCSDKTensorFilterTestHelper.cc]: need a space

File: tests/nnstreamer_filter_mvncsdk2/unittest_filter_mvncsdk2.cc [hunk 2]
```diff
@@ -90,6 +90,85 @@ TEST (pipeline_mvncsdk2_filter, launch_normal)
...
+            (ret == GST_STATE_CHANGE_SUCCESS)) { \
+          test = true;\
+        } \
+        EXPECT_NE (test, true); \
+        EXPECT_EQ (ret, GST_STATE_CHANGE_FAILURE); \
+        \
+        gst_object_unref (gstpipe); \
+      } else { \
+        status = -1; \
+        g_printerr("Failed to launch the pipeline, %s : %s\n", pipeline, \
```
Comment (dongju-chae) [inline tests/nnstreamer_filter_mvncsdk2/unittest_filter_mvncsdk2.cc]: need a space

File: tests/nnstreamer_filter_mvncsdk2/NCSDKTensorFilterTestHelper.cc [hunk 3]
```diff
@@ -109,13 +130,48 @@ NCSDKTensorFilterTestHelper::release ()
...
  */
 ncStatus_t
 NCSDKTensorFilterTestHelper::ncGlobalGetOption (int option, void *data,
       unsigned int *dataLength)
 {
+  if (this->mFailStage == fail_stage_t::WRONG_SDK_VER) {
+    ncsdk_ver_t ver;
+
+    // MAJOR version number should be 2
+    ver[MAJOR] = 3;
```
Comment (dongju-chae) [inline tests/nnstreamer_filter_mvncsdk2/NCSDKTensorFilterTestHelper.cc]: It seems like a mismatch with the above comment.

Comment (dongju-chae) [reply tests/nnstreamer_filter_mvncsdk2/NCSDKTensorFilterTestHelper.cc]: My bad :(

**Comment by wooksong** [comment]
#### V3
- @dongju-chae's comments are applied.

File: tests/nnstreamer_filter_mvncsdk2/NCSDKTensorFilterTestHelper.cc (hunk 1 above)
Comment (wooksong) [reply tests/nnstreamer_filter_mvncsdk2/NCSDKTensorFilterTestHelper.cc]: Thnx!

File: tests/nnstreamer_filter_mvncsdk2/unittest_filter_mvncsdk2.cc (hunk 2 above)
Comment (wooksong) [reply tests/nnstreamer_filter_mvncsdk2/unittest_filter_mvncsdk2.cc]: Thnx!

**Pull request merged**